docs/UserGuide.adoc

```diff
@@ -86,44 +86,49 @@ Format: `help`
 [TIP]
 Optionally, you can press kbd:[F1] or click `help` which can be found on the menu bar at the top left hand corner of the application.
 
+The following image shows how the help bar can be accessed:
+
+[.text-center]
 .Accessing Help from Menu Bar
 image::help.png[width="300"]
 
-=== Flexible date format
+Help can be accessed from the menu bar by clicking on the `Help` button.
+
+=== Flexible date formats
 
 Whenever a particular command requires a `DATE` field, you can choose either of the following formats.
 
-* Formal Dates
+* *Formal Dates*
 ** 1978-01-28
 ** 1984/04/02
 ** 1/02/1980
 ** 26/2/2019
-* Relaxed Dates
+* *Relaxed Dates*
 ** The 31st of April in the year 2008
 ** Fri, 21 Nov 1997
 ** Jan 21,'97
 ** jan 1st
 ** february twenty-eight
-* Relative Dates
+* *Relative Dates*
 ** next thursday/ last wednesday
 ** today/ tomorrow/ yesterday
 ** next week/ next month / next year
 ** 3 days from now
 ** three weeks ago
-* Date Alternatives
+* *Date Alternatives*
 ** next wed or thurs
 ** oct 3rd or 4th
-* Prefixes
+* *Prefixes*
 ** day after/ the day before
 ** the monday after/ the monday before
 ** 2 fridays before/ 4 tuesdays after
 
 [NOTE]
-You do not need to specify which date format you would like to use as the application will be able to differentiate it from the input received.
+You do not need to specify which date format you would like to use as MoneyGoWhere will be able to differentiate it from the input received.
 
 [NOTE]
-The application will also automatically check whether a given date is valid.
-Should you accidentally provide an invalid date such as 31/02/2019, it will be rejected as there are only 28 days in February 2019.
+MoneyGoWhere will also automatically check whether a given date is valid.
+Should you accidentally provide an invalid date such as `31/02/2019`, it will be rejected as there are only 28 days in February 2019.
```

Comment:
`31/02/2019` should not be highlighted as this format is reserved for command key words as specified in section 2, How to use this guide.

Reply:
Changed!